FirebaseAuth/Sources/Auth/FIRAuth.m

```diff
@@ -2307,7 +2294,7 @@ - (nullable NSString *)getUserID {
 - (BOOL)internalUseUserAccessGroup:(NSString *_Nullable)accessGroup
                              error:(NSError *_Nullable *_Nullable)outError {
   BOOL success;
-  success = [self.storedUserManager setStoredUserAccessGroup:accessGroup error:outError];
```

Comment:
Isn't the `outError` necessary to set the error code in the public api: `- (BOOL)useUserAccessGroup:(NSString *_Nullable)accessGroup
                     error:(NSError *_Nullable *_Nullable)outError`

Reply:
If you look at the implementation of that API, no code paths set that error code.

Reply:
The problem is that AuthUserDefaults conforms to AuthStorage– which defines throwing API for getting and setting values. But, the implementation of AuthUserDefaults does not actually throw any errors. So clients that call this code have error handling code paths that are effectively no-ops. We could change it so that instead of returning `nil` when, for example, nothing is stored, we return an error–– but that could get risky as there could be some subtle behavior changes (I'd have to think about it more).

Reply:
I see. thanks for the clarification.